dapp/e2e/governance.spec.ts

```diff
@@ -0,0 +1,11 @@
+import { test } from "@playwright/test";
+
+test("Submit proposal", async ({ page }) => {
+  await page.goto("/governance?name=tansu");
+  await page.getByRole("button", { name: "Submit proposal" }).click();
+});
+
+test("Go to project detail page", async ({ page }) => {
+  await page.goto("/governance?name=tansu");
+  await page.getByText("0Add a DAO systemCancelled").click();
```

Comment:
Note that this will fail when there is going to be a network reset. Or if we test in a docker container. (Which ideally we should be able to do locally. Locally we can spin up the whole stack in a compose and run tests there.)